packages/core/src/navigation/navigation.element.scss

```diff
@@ -65,3 +65,7 @@
   height: 100%;
   overflow: auto;
 }
+
+::slotted([slot='cds-navigation-substart']) {
```

Comment:
Small fix for something I must have broke whenI was cleaning up the shadowDOM and css.